netci.groovy

```diff
@@ -117,13 +117,13 @@ def CreateStyleCheckTasks = { taskString, taskName, checkName ->
         def jobName = Utilities.getFullJobName(project, taskName, isPR)
 
         def newJob = job(jobName) {
-            label('ubuntu')
             steps {
                 shell(taskString)
             }
         }
 
         Utilities.simpleInnerLoopJobSetup(newJob, project, isPR, checkName)
+        Utilities.setMachineAffinity(newJob, 'Ubuntu14.04', 'latest-or-auto')
```

Comment:
We used to specify just plain "ubuntu' but now there is 15.10 so we avoid confusion by eventually removing the original label and adding Ubuntu14.04.  The latest-or-auto bit specifies that the latest image should be used.  In this case it's a dynamically created VM.  Verified that the command works on those VMs